rusk-wallet/src/bin/command.rs

```diff
@@ -194,6 +194,10 @@ pub(crate) enum Command {
         #[arg(short, long)]
         address: Option<Address>,
 
+        /// Amount of rewards to withdraw from the stake contract
+        #[arg(short, long)]
+        reward: Dusk,
```

Comment:
I disagree, if the person isnt aware they can specify the amount and they dont want to withdraw all of it then it will be a problem, will leave the user confused. Even if we document this behavior people arent gonna read it. I can add option to withdraw all but they will have to specify if _they want to do so_

Reply:
I would say that it's most likely that the user wants to withdraw all the funds so having this behavior activated by default makes sense to me.
And if not all funds should be withdrawn, then the user can specify so we don't sacrifice any functionality with my proposed solution.
We do however add friction with your suggestion since the user now needs to be aware and check the max funds available whenever she wants to retrieve the reward.

The case that the user is not aware of certain functionalities shouldn't guide our design decisions.

Reply:
@Daksh14 I agree with Moana here. The approach she suggests would lead to the least amount of friction. Users will always have to end up requesting for their reward balance, even though the vast majority will want to withdraw their full reward balance anyway.

We should always design with Pareto principle in mind. Focus on the needs of the majority (80%), but make sure the minority (20%) can still achieve their goals

Reply:
Amt is now optional and if its none then all the available rewards will be withdrawn @HDauven @moCello